src/FileDownloader.java

```diff
@@ -182,7 +102,7 @@ private static void downloadFile() {
                     FileOutputStream fos = new FileOutputStream(dir + fileName);
                     ProgressBarThread progressBarThread = new ProgressBarThread(fos, totalSize, fileName);
                     progressBarThread.start();
-                    Drifty_CLI.logger.log("INFO", "Downloading " + fileName + " ...");
+                    Drifty_CLI.logger.log(LOGGER_INFO, DOWNLOADING + fileName + " ...");
```

Comment:
I think adding a constant with name `DOWNLOADING` is no better than just keeping it as a `Downloading `(string).
```suggestion
                    Drifty_CLI.logger.log(LOGGER_INFO, "Downloading " + fileName + " ...");
```